### PR TITLE
Translated React Without ES6

### PR DESCRIPTION
**Pull request by lunmay**

### CODE REVIEW

File: content/docs/react-without-es6.md [hunk 1]
```diff
@@ -1,54 +1,54 @@
...
+    return <h1>Halo {this.props.name}</h1>;
   }
 });
 ```
 
-The API of ES6 classes is similar to `createReactClass()` with a few exceptions.
+API untuk kelas ES6 mirip dengan `createReactClass()`, dengan beberapa pengecualian.
 
-## Declaring Default Props {#declaring-default-props}
+## Mendeklarasikan *Props* *Default* {#declaring-default-props}
```
Comment (regalius) [inline content/docs/react-without-es6.md]: Loanwords in the heading doesn't have to be italicized 😄

File: content/docs/react-without-es6.md [hunk 2]
```diff
@@ -82,16 +82,16 @@ var Counter = createReactClass({
 });
 ```
 
-## Autobinding {#autobinding}
+## *Autobinding* {#autobinding}
 
-In React components declared as ES6 classes, methods follow the same semantics as regular ES6 classes. This means that they don't automatically bind `this` to the instance. You'll have to explicitly use `.bind(this)` in the constructor:
+Pada komponen React yang dideklarasikan sebagai kelas ES6, *method* mengikuti semantik yang sama seperti kelas ES6 reguler. Ini berarti *method* tersebut tidak secara otomatis mem-*bind* `this` kepada instans. Anda harus secara eksplisit menggunakan `.bind(this)` pada konstruktor:
```
Comment (regalius) [inline content/docs/react-without-es6.md]: instans -> *instance*

File: content/docs/react-without-es6.md [hunk 3]
```diff
@@ -125,61 +126,61 @@ var SayHello = createReactClass({
...
 ```javascript
 class SayHello extends React.Component {
   constructor(props) {
     super(props);
-    this.state = {message: 'Hello!'};
+    this.state = {message: 'Halo!'};
   }
-  // WARNING: this syntax is experimental!
-  // Using an arrow here binds the method:
+  // PERINGATAN: sintalsis ini bersifat eksperimental!
```
Comment (regalius) [inline content/docs/react-without-es6.md]: `sintaksis` typo I think 😄

File: content/docs/react-without-es6.md [hunk 4]
```diff
@@ -125,61 +126,61 @@ var SayHello = createReactClass({
...
+## Mixin {#mixins}
 
->**Note:**
+>**Catatan:**
 >
->ES6 launched without any mixin support. Therefore, there is no support for mixins when you use React with ES6 classes.
+>ES6 diluncurkan tanpa adanya dukungan *mixin*. Oleh sebab itu, tidak ada dukungan untuk *mixin* saat Anda menggunakan React dengan kelas ES6.
 >
->**We also found numerous issues in codebases using mixins, [and don't recommend using them in the new code](/blog/2016/07/13/mixins-considered-harmful.html).**
+>**Kami juga menemukan banyak masalah pada kode dasar yang menggunakan *mixin*, [dan tidak menyarankan untuk menggunakannya untuk kode yang baru](/blog/2016/07/13/mixins-considered-harmful.html).**
```
Comment (regalius) [inline content/docs/react-without-es6.md]:
```suggestion
>**Kami juga menemukan banyak masalah pada basis kode yang menggunakan *mixin*, [dan tidak menyarankan untuk menggunakannya untuk kode yang baru](/blog/2016/07/13/mixins-considered-harmful.html).**
```

What do you think? 😄

File: content/docs/react-without-es6.md [hunk 5]
```diff
@@ -222,4 +223,4 @@ ReactDOM.render(
 );
 ```
 
-If a component is using multiple mixins and several mixins define the same lifecycle method (i.e. several mixins want to do some cleanup when the component is destroyed), all of the lifecycle methods are guaranteed to be called. Methods defined on mixins run in the order mixins were listed, followed by a method call on the component.
+Jika komponen menggunakan beberapa *mixin* dan beberapa *mixin* mendefinisikan *lifecycle method* yang sama (beberapa *mixin* ingin melakukan pembersihan ketika komponen dihancurkan), semua *lifecycle method* dijamin untuk dipanggil. *Method* yang didefiniskan pada *mixin* dijalankan secara berurutan sesuai dengan dengan proses pendaftarannya, diikuti dengan pemanggilan *method* pada komponen.
```
Comment (regalius) [inline content/docs/react-without-es6.md]:
```suggestion
Jika komponen menggunakan beberapa *mixin* dan beberapa *mixin* mendefinisikan *lifecycle method* yang sama (contohnya, beberapa *mixin* ingin melakukan pembersihan ketika komponen dihancurkan), semua *lifecycle method* dijamin untuk dipanggil. *Method* yang didefinisikan pada *mixin* dijalankan secara berurutan sesuai dengan dengan proses pendaftarannya, diikuti dengan pemanggilan *method* pada komponen.
```

Several typos I think 😄

File: content/docs/react-without-es6.md (hunk 1 above)
Comment (grikomsn) [inline content/docs/react-without-es6.md]:
```suggestion
## Mendeklarasikan Props Default {#declaring-default-props}
```

File: content/docs/react-without-es6.md [hunk 6]
```diff
@@ -1,54 +1,54 @@
...
 }
 
 Greeting.defaultProps = {
-  name: 'Mary'
+  name: 'Maria'
 };
 ```
 
-With `createReactClass()`, you need to define `getDefaultProps()` as a function on the passed object:
+Dengan `createReactClass()`, Anda harus mendefinisikan `getDefaultProps()` sebagai fungsi pada objek yang dioperkan:
```
Comment (grikomsn) [inline content/docs/react-without-es6.md]: Which one sounds better? `dioperkan` or `diberikan`?

```suggestion
Dengan `createReactClass()`, Anda harus mendefinisikan `getDefaultProps()` sebagai fungsi pada objek yang diberikan:
```

File: content/docs/react-without-es6.md [hunk 7]
```diff
@@ -57,9 +57,9 @@ var Greeting = createReactClass({
 });
 ```
 
-## Setting the Initial State {#setting-the-initial-state}
+## Menyetel *State* Awal {#setting-the-initial-state}
```
Comment (grikomsn) [inline content/docs/react-without-es6.md]:
```suggestion
## Menyetel State Awal {#setting-the-initial-state}
```

File: content/docs/react-without-es6.md [hunk 8]
```diff
@@ -82,16 +82,16 @@ var Counter = createReactClass({
 });
 ```
 
-## Autobinding {#autobinding}
+## *Autobinding* {#autobinding}
```
Comment (grikomsn) [inline content/docs/react-without-es6.md]:
```suggestion
## Autobinding {#autobinding}
```

File: content/docs/react-without-es6.md (hunk 2 above)
Comment (grikomsn) [inline content/docs/react-without-es6.md]:
```suggestion
Pada komponen React yang dideklarasikan sebagai kelas ES6, *method* mengikuti semantik yang sama seperti kelas ES6 reguler. Ini berarti *method* tersebut tidak secara otomatis mem-*bind* `this` kepada *instance*. Anda harus secara eksplisit menggunakan `.bind(this)` pada konstruktor:
```

File: content/docs/react-without-es6.md (hunk 3 above)
Comment (grikomsn) [inline content/docs/react-without-es6.md]:
```suggestion
  // PERINGATAN: sintaksis ini bersifat eksperimental!
```

File: content/docs/react-without-es6.md [hunk 9]
```diff
@@ -125,61 +126,61 @@ var SayHello = createReactClass({
...
     );
   }
 }
 ```
 
-Please note that the syntax above is **experimental** and the syntax may change, or the proposal might not make it into the language.
+Perhatikan bahwa sintaksis di atas bersifat **eksperimental** dan sintaksis mungkin akan berubah, atau proposal ini tidak akan disertakan dalam bahasa.
 
-If you'd rather play it safe, you have a few options:
+Jika Anda ingin bermain pada area yang aman, Anda memiliki beberapa opsi:
```
Comment (grikomsn) [inline content/docs/react-without-es6.md]:
```suggestion
Jika Anda ingin bermain aman, Anda memiliki beberapa opsi:
```

Also, should idioms like `play it safe` be translated as is?

File: content/docs/react-without-es6.md [hunk 10]
```diff
@@ -125,61 +126,61 @@ var SayHello = createReactClass({
...
 
-* Bind methods in the constructor.
-* Use arrow functions, e.g. `onClick={(e) => this.handleClick(e)}`.
-* Keep using `createReactClass`.
+* Mem-*bind* *method* dalam konstruktor.
+* Gunakan *arrow function*, misalnya `onClick={(e) => this.handleClick(e)}`.
+* Terus gunakan `createReactClass`.
```
Comment (grikomsn) [inline content/docs/react-without-es6.md]:
```suggestion
* Tetap menggunakan `createReactClass`.
```

File: content/docs/react-without-es6.md (hunk 4 above)
Comment (grikomsn) [reply content/docs/react-without-es6.md]: LGTM 👍

**Pull request merged**